course_discovery/apps/course_metadata/data_loaders.py

```
@@ -127,11 +124,12 @@ class OrganizationsApiDataLoader(AbstractDataLoader):
     """ Loads organizations from the Organizations API. """
 
     def ingest(self):
-        client = self.api_client
+        api_url = self.partner.organizations_api_url
```

Yeah, probably, although I'm not sure if we should raise an exception and exit, or if we should write to the log and continue -- thoughts on that?